templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -12,7 +12,7 @@
     "import pandas as pd\n",
```

Comment:
<p>One small change here, I think giving the labeled output a more informative name (cluster_labels_xr, or something like that) would be useful to help distinguish the many different xarrays that will be produced. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
changed